### DISCUSSION TITLE
Problem with the storage of file FileUpload

### DISCUSSION THREAD

**Discussion by benjamin-breuil** [filamentphp/filament#8027] [Help]
My problem is that when I add an image to a form, the image link is saved in the database, the image is put in storage/app/public/images except that I did php artisan storage:link and the image is not added in public/storage/images/.

PS C:\Users\BB\Documents\zermapp-developement\zermapp> php artisan storage:link

   ERROR  The [C:\Users\BB\Documents\zermapp-developement\zermapp\public\storage] link already exists.

in .env

APP_URL=http://localhost:8000

**Comment by leandrocfe** [comment]
what is the issue?

**Reply by benjamin-breuil** [reply]
My problem is that when I add an image to a form, the image link is saved in the database, the image is put in storage/app/public/images except that I've done php artisan storage:link and the image isn't added to public/storage/images/ and when I go back to rmon entered to modify it the image isn't displayed and there's just the name of the image so I can't retrieve an image with a link example: http://localhost/public/storage/images/images.png doesn't work
